packages/server/lib/modes/run.ts

```diff
@@ -746,7 +781,25 @@ async function runSpecs (options: { config: Cfg, browser: Browser, sys: any, hea
     autoCancelAfterFailures,
   }
 
+  const runSpan = telemetry.startSpan({ name: 'run' })
+
+  runSpan?.setAttributes({
+    record: !!runUrl,
```

Comment:
more of a nit. I think we reference `record` as a boolean elsewhere
```suggestion
    recordEnabled: !!runUrl,
```